questionpy_sdk/webserver/context.py

```diff
@@ -0,0 +1,92 @@
+#  This file is part of QuestionPy. (https://questionpy.org)
+#  QuestionPy is free software released under terms of the MIT license. See LICENSE.md.
+#  (c) Technische Universität Berlin, innoCampus <info@isis.tu-berlin.de>
+
+from typing import List, Any, Optional, Dict
+
+from questionpy_common.elements import StaticTextElement, CheckboxElement, TextInputElement, FormElement, \
+    GroupElement, RepetitionElement, OptionsFormDefinition, CheckboxGroupElement, RadioGroupElement, SelectElement, \
+    HiddenElement
+
+from questionpy_sdk.webserver.elements import CxdOptionsFormDefinition, CxdFormSection, CxdFormElement, \
+    CxdGroupElement, CxdRepetitionElement, CxdStaticTextElement, CxdTextInputElement, \
+    CxdCheckboxElement, CxdCheckboxGroupElement, CxdRadioGroupElement, CxdSelectElement, CxdHiddenElement
+
+element_mapping: Dict[type, type] = {
+    StaticTextElement: CxdStaticTextElement,
+    TextInputElement: CxdTextInputElement,
+    CheckboxElement: CxdCheckboxElement,
+    CheckboxGroupElement: CxdCheckboxGroupElement,
+    RadioGroupElement: CxdRadioGroupElement,
+    SelectElement: CxdSelectElement,
+    HiddenElement: CxdHiddenElement
+}
+
+
+def _contextualize_element(element: FormElement, form_data: Optional[dict[str, Any]], path: List[str],
+                           context: Optional[dict] = None) -> CxdFormElement:
+    path.append(element.name)
+    element_form_data = None
+    if form_data:
+        element_form_data = form_data.get(element.name)
+
+    if isinstance(element, GroupElement):
+        cxd_gr_element = CxdGroupElement(path=path.copy(), **element.model_dump(exclude={'elements'}))
+        cxd_gr_element.cxd_elements = _contextualize_element_list(element.elements, element_form_data, path, context)
+        path.pop()
+        return cxd_gr_element
+
+    if isinstance(element, RepetitionElement):
+        cxd_rep_element = CxdRepetitionElement(path=path.copy(), **element.model_dump(exclude={'elements'}))
+        if not element_form_data:
+            for i in range(element.initial_repetitions):
+                path.append(str(i))
+                element_list = _contextualize_element_list(element.elements, None, path, {'repno': i})
+                cxd_rep_element.cxd_elements.append(element_list)
+                path.pop()
+            path.pop()
+            return cxd_rep_element
+        for i, repetition in enumerate(element_form_data):
+            path.append(str(i))
+            element_list = _contextualize_element_list(element.elements, repetition, path, {'repno': i})
+            cxd_rep_element.cxd_elements.append(element_list)
+            path.pop()
+        path.pop()
+        return cxd_rep_element
+
+    if element.__class__ not in element_mapping:
```

Comment:
```suggestion
    if type(element) not in element_mapping:
```

Heutzutage kein Unterschied, aber konventioneller

Reply:
Magst du das auch an den anderen Stellen übernehmen? Z.b. 3 Zeilen drunter?